modules/core/include/opencv2/core/types.hpp

```diff
@@ -181,8 +181,8 @@ template<typename _Tp> class Point_
     double cross(const Point_& pt) const;
     //! checks whether the point is inside the specified rectangle
     bool inside(const Rect_<_Tp>& r) const;
-
-    _Tp x, y; //< the point coordinates
+    //! the point coordinates
+    _Tp x, y;
```

Comment:
Original same-line comment should be "//!<" (missing '!')

See the same change below (line 266)